sql/core/src/test/scripts/gen-avro.sh

```diff
@@ -1,3 +1,5 @@
+#!/usr/bin/env bash
+
```

Comment:
SC2148 (error): Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.